code/__HELPERS/unsorted.dm

```diff
@@ -1210,6 +1210,9 @@ proc/pick_closest_path(value, list/matches = get_fancy_list_of_atom_types())
 #define DELTA_CALC max(((max(world.tick_usage, world.cpu) / 100) * max(Master.sleep_delta,1)), 1)
 
 /proc/stoplag()
+	if (!Master || !(Master.current_runlevel & RUNLEVELS_DEFAULT))
```

Comment:
No, This works better because its living.

The loop sleep system was designed to prevent contention among alot of things wanting to do shit at once, because sleep has a fairly costly overhead, so sleeping, waking up, only to sleep again because the tick is already gone, is wasteful.

This edge case is generally only likely to occur when subsystems are running, so tying this to the same define that controls when most subsystems run makes senses.